app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1398,8 +1400,12 @@ class MultipleDisposalsTriageController @Inject() (
       }
     }
 
-  def isAValidCGTTaxTear(taxYearExchanged: TaxYearExchanged): Boolean =
-    !(taxYearExchanged === TaxYearExchanged.TaxYearBefore2020 || taxYearExchanged === TaxYearExchanged.DifferentTaxYears)
+  def isAValidCGTTaxTear(taxYearExchanged: TaxYearExchanged, originalSubmissionYear: Option[String] = None): Boolean =
+    TaxYearExchanged.taxYearsMap.get(originalSubmissionYear.getOrElse("")) match {
+      case Some(x) => x === taxYearExchanged
+      case None    =>
+        !(taxYearExchanged === TaxYearExchanged.TaxYearBefore2020 || taxYearExchanged === TaxYearExchanged.DifferentTaxYears)
+    }
 
   def checkYourAnswersSubmit(): Action[AnyContent] =
```

Comment:
You are using existing "exit page" --> /exit/different-tax-year
We should build new exit page, please check prototype